solr/core/src/java/org/apache/solr/core/SolrCore.java

```diff
@@ -1238,12 +1241,11 @@ private SolrCore(
   }
 
   public void seedVersionBuckets() {
-    UpdateHandler uh = getUpdateHandler();
-    if (uh != null && uh.getUpdateLog() != null) {
+    if (updateHandler != null && updateHandler.getUpdateLog() != null) {
       RefCounted<SolrIndexSearcher> newestSearcher = getRealtimeSearcher();
       if (newestSearcher != null) {
         try {
-          uh.getUpdateLog().seedBucketsWithHighestVersion(newestSearcher.get());
+          updateHandler.getUpdateLog().seedBucketsWithHighestVersion(newestSearcher.get());
```

Comment:
This change is not relevant to the main goal of the PR, it would clarify things if it these lines were left unchanged.